test/framework/src/main/java/org/elasticsearch/test/transport/MockTransportService.java

```diff
@@ -688,4 +693,12 @@ public void close() throws IOException {
             connection.close();
         }
     }
+
+    public Transport getDelegateTransport() {
```

Comment:
nit: I think this method name is confusing, since it does *not* return a DelegateTransport. Perhaps `getOriginalTransport()` or `getRealTransport()` or something like that?